mitxpro/templates/base.html

```diff
@@ -18,6 +18,9 @@
     <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css" rel="stylesheet"/>
     <script type="text/javascript">
     var SETTINGS = {{ js_settings_json|safe }};
+    {% if CSOURCE_PAYLOAD %}
+        var CSOURCE_PAYLOAD = {{ CSOURCE_PAYLOAD|safe }};
```

Comment:
Can we define this variable as `null` if `CSOURCE_PAYLOAD` does not exist?

Reply:
Updated